frontend/src/context/auth.tsx

```diff
@@ -28,7 +28,7 @@ const AuthProvider: FC<DefaultOptions> = ({ children }) => {
 
   useEffect(() => {
     localStorage.setItem('auth', JSON.stringify(auth));
```

Comment:
Lets take note to use secure cookie. LGTM 👍🏽

Reply:
Got it, we'll figure it out once we integrate the real api